aviator/servicelist.go

```
@@ -135,16 +140,10 @@ func (s *ServiceList) shutdown() error {
 		}
 		pid := service.cmd.Process.Pid
 		if isProcessRunning(pid) {
-			// Kill the process
-			err := service.cmd.Process.Kill()
-			if err != nil {
-				return err
-			}
-			// Wait for it to actually die
-			_, err = service.cmd.Process.Wait()
-			if err != nil {
-				return err
-			}
+			// Sending a kill signal to -pid kills the process group
+			// for the process pid
+			syscall.Kill(-pid, syscall.SIGKILL)
+			service.cmd.Process.Wait()
```

It was safe before as well.  This only throws an error for a race condition (the process existed when we did isProcessRunning, but died before we sent the kill signal), in which case we can ignore the error.